scripts/dfx-asset-sources.sh

```diff
@@ -11,10 +11,16 @@ MOTOKO_X86_64_DARWIN_SHA256="c3da662922f5e57915c4488c9252bc4054a8e9c6c514ccb6197
 MOTOKO_X86_64_DARWIN_URL="https://github.com/dfinity/motoko/releases/download/0.6.20/motoko-macos-0.6.20.tar.gz"
 MOTOKO_X86_64_LINUX_SHA256="a2caca3401bc3e521d44c10eb3bc094e6b8d441c6bd77809809a2bdc6055823b"
 MOTOKO_X86_64_LINUX_URL="https://github.com/dfinity/motoko/releases/download/0.6.20/motoko-linux64-0.6.20.tar.gz"
+# The replica and canister_sandbox binaries must have the same revision.
```

Comment:
If you want these comments to be in this file, scripts/write-dfx-asset-sources.sh will have to output them.